includes/class-sensei-question.php

```diff
@@ -923,7 +923,7 @@ public static function the_answer_feedback( $question_id ) {
 		$quiz_graded        = $user_quiz_progress && ! in_array( $user_quiz_progress->get_status(), array( 'ungraded', 'in-progress' ) );
 
 		$quiz_required_pass_grade = intval( get_post_meta( $quiz_id, '_quiz_passmark', true ) );
-		$succeeded                = $user_quiz_grade >= $quiz_required_pass_grade;
+		$succeeded                = ! Sensei_Quiz::is_pass_required( $lesson_id ) || $user_quiz_grade >= $quiz_required_pass_grade;
```

Comment:
In the UI, we hide the "Passing Grade" if the "Pass Required" is turned off. So the behavior in the frontend should be the same ("Passing Grade" shouldn't matter if "Pass Required" is turned off).

In this case, we are considering the quiz as succeeded if "Pass Required" is turned off for the answer feedback logic.